pkg/controller/storagecluster/reconcile.go

```diff
@@ -903,6 +904,22 @@ func (r *ReconcileStorageCluster) deleteResources(sc *ocsv1.StorageCluster, reqL
 	return r.deleteNoobaaSystems(sc, reqLogger)
 }
 
+//getCephDaemonPlacements returns placement configuration for ceph components with appropriate topology
+func getCephDaemonPlacements(sc *ocsv1.StorageCluster, component string) rook.Placement {
+	topologyMap := sc.Status.NodeTopologies
```

Comment:
Query: What should be the behavior when the topologyMap is not available? (I believe it would happen in non AWS environments)

Reply:
when there is no Topology Map, default placement should be used. If we want to be more flexible, we should allow custom placements to user.